docs/docsite/rst/command_guide/index.rst

```diff
@@ -18,4 +18,11 @@ Ansible provides ad hoc commands and several utilities for performing various op
 
    intro_adhoc
    command_line_tools
-   cheatsheet
\ No newline at end of file
+   cheatsheet
+
+.. seealso::
+
+   `Ansible Navigator <https://ansible.readthedocs.io/projects/navigator/>`_
+       A command-line tool and a TUI that provides a convenient user interface for most
+       of the native Ansible command-line utilities and allows to run Ansible automation content
+       inside containers (:ref:`execution environments <getting_started_ee_index>`)
```

Comment:
Shouldn't this have a period at the end?
```suggestion
       inside containers (:ref:`execution environments <getting_started_ee_index>`).
```

Reply:
@felixfontein @samccann i thought of the same. the issue is that it doesn't end with a period on other pages in seealso's. We should be consistent across the site. Let's merge the PR as is if no objections and discuss the "use or not to use dots" topic separately, what do you think?

Reply:
Unfortunately, faik this ref to the getting started won't work because it's only in the Package version of the docs, not in the core version. Can you change it to a direct link for now so we can merge this PR?

I've created https://github.com/ansible/ansible-documentation/issues/311 to track the places we need to change in a separate PR for intersphinx links updates

Reply:
sure thing, done, PTAL